website/versioned_docs/version-2.1.0/prysm-usage/slasher.md

```diff
@@ -18,7 +18,7 @@ These specifications must be met in order to successfully run a Prysm beacon nod
 * Operating System: 64-bit Linux, Mac OS X 10.14+, Windows 64-bit
 * Processor: Intel Core i5–760 or AMD FX-8100 or better
 * Memory: 8GB RAM
-* Storage: 1TB available space SSD
+* Storage: 200GB available space SSD
```

Comment:
is this accurate? 🤔

Reply:
updated ty! (todo: improve the way we do versioning / syncing between version directories in docusaurus....)